JeMPI_Apps/JeMPI_UI/src/components/shell/NavigationBar.tsx

```diff
@@ -140,7 +140,7 @@ const NavigationBar = () => {
               width: { xs: '100%', md: 'auto' }
             }}
           >
-            <JembiLogo />
+            {/* <JembiLogo /> */}
```

Comment:
Oh, I thought we were doing this on a different branch (not jempi-base) ? Do we want to have the logo on our base branch. An option could be to make the a configuration point which can be updated per client? @ajinkyagadewar any thoughts on this?

Reply:
actually, there is a new commit that adds an environment  variable to disable or enable Jempi brand 
https://github.com/jembi/JeMPI/pull/121/commits/6e7e698b8485b0ccd12f73adb5e9579ddf8b7784